ansible/simple/list_zones.yml

```diff
@@ -0,0 +1,11 @@
+---
+- hosts: localhost
+  tasks:
+  - name: Collect zones for Pure Storage
+    purestorage.fusion.fusion_info:
+      gather_subset: zones
```

Comment:
Is this Availability Zones? If it is, can we name the task as Collect Availability Zones?